app/helpers/time_helper.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+module TimeHelper
+  def date_range(from_date, until_date, options = {})
+    options.symbolize_keys!
+    format = options[:format] || :long
+    separator = options[:separator] || "—"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.